bot.py

```diff
@@ -81,7 +81,8 @@ def on_message(message):
             print(str(subprocess.Popen('git pull', shell=True, stdout=subprocess.PIPE).stdout.read()))
             yield from client.send_message(message.channel, 'Update Successful! Restarting...')
             # Restart
-            os.system('python3 launch.py')
```

Comment:
I believe that `os.system` will wait for the command to terminate before returning. `subprocess.Popen` will not wait.